apps/dapp/src/components/Layouts/CoreLayout/WrongNetworkPopover.tsx

```diff
@@ -47,7 +47,7 @@ export const WrongNetworkPopover = () => {
     }
 
     const isSupported = isSupportedChain(currentNetworkId);
-    if ((!isSupported && !isOpen) || (!IS_PROD && defaultChainForEnv.id !== currentNetworkId)) {
+    if ((!isSupported && !isOpen) || (defaultChainForEnv.id !== currentNetworkId)) {
```

Comment:
this was causing not to show modal when `VITE_ENV=production`

Reply:
@nichosystem could you confirm if this change is reasonable? not sure why it was initially there